src/Symfony/Component/Runtime/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "symfony/runtime",
+    "type": "library",
+    "description": "Enables decoupling PHP apps from global state",
+    "homepage": "https://symfony.com",
+    "license" : "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5"
+    },
+    "require-dev": {
+        "symfony/console": "^4.4|^5",
+        "symfony/dotenv": "^5.1",
+        "symfony/http-foundation": "^4.4|^5",
+        "symfony/http-kernel": "^4.4|^5"
+    },
+    "conflict": {
+        "symfony/dotenv": "<5.1"
```

Comment:
My current Symfony 5.0 applications have a bootstrap.php file. I guess that file is not needed when using dotenv 5.1, is that correct?

Reply:
Correct, the recipes for 5.1 removed it.